ludwig/schema/__init__.py

```diff
@@ -81,8 +81,8 @@ def custom_is_array(checker, instance):
     return extend(Draft7Validator, type_checker=type_checker)
 
 
-@DeveloperAPI
-def validate_config(config):
+@Deprecated(message="Use 'from ludwig.config_validation import validate_config' instead.")
```

Comment:
Great call deprecating instead of removing outright!

Reply:
+1